backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/navigationbar/BpkTopNavBar.kt

```diff
@@ -51,14 +59,16 @@ fun BpkTopNavBar(
     modifier: Modifier = Modifier,
     insets: WindowInsets? = WindowInsets.statusBars,
     actions: List<IconAction> = emptyList(),
+    style: NavBarStyle = NavBarStyle.Default,
```

Comment:
API alignment – this wasn't included to fixed TopNavBar

Reply:
only the expanded one has the transparent style, so it wasn't added intentionally